dokka-integration-tests/gradle/src/testExternalProjectKotlinxCoroutines/kotlin/CoroutinesGradleIntegrationTest.kt

```diff
@@ -53,7 +53,7 @@ class CoroutinesGradleIntegrationTest : AbstractGradleIntegrationTest(), TestOut
         val result = createGradleRunner(
             buildVersions,
             ":dokkaHtmlMultiModule", "-i", "-s",
-            jvmArgs = listOf("-Xmx2G", "-XX:MaxMetaspaceSize=500m")
+            jvmArgs = listOf("-Xmx2G", "-XX:MaxMetaspaceSize=2g")
```

Comment:
Is there a specific reason for increasing metaspace size? These tests also helped us find metaspace memory leaks a couple of times

Reply:
I increased it because the test failed :) I didn't realise it was intentionally low - I'll set it back and add a comment to clarify why it should be kept low.